.github/workflows/CI.yml

```diff
@@ -10,7 +10,7 @@ on:
     branches:
       - 'master'
       - 'release-*'
-    type: [labeled]
+    types: [labeled]
```

Comment:
@jsoref I think you found a bug: `type: [labeled]` never worked. Consequently, your change to `types` fixes the behavior but, it just triggers the action when a PR has a label attached. By default, we don't put labels on a PR. 

Can you please change it to: `types: [labeled, unlabeled]`

Reply:
I'm pretty sure you don't want that either. Speaking from experience, the GitHub documentation is immensely frustrating...

https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request

I suspect you want this to be omitted entirely.

`type: [labeled]` would result in only processing pull requests when they gain a label.
`type: [unlabeled]` would result in only processing pull requests when they lose a label.
`type: [labeled, unlabeled]` would result in only processing pull requests when they gain or lose a label.

The default is `type: [opened, synchronize, reopened]`.

You could conceivably want to add `labeled` to that list. But, it's more likely that you just want the line removed so you get the default (which is what you have now).

lmk, and I'll adjust.

Reply:
```suggestion
    types: [opened, synchronize, reopened, labeled, unlabeled]
```

Reply:
To make it explicit when the action runs, I like your suggestion with: `types: [opened, synchronize, reopened, labeled, unlabeled]`

Can you please adapt your PR accordingly? (I can`t commit your suggestion, because  I don`t get it signed for DCO)

Reply:
@johannes-b: done

Reply:
Thanks!